front/src/modules/settings/developers/constants/expirationDates.ts

```diff
@@ -0,0 +1,14 @@
+import { IconComponent } from '@/ui/display/icon/types/IconComponent';
+
+export const ExpirationDates: {
+  value: number;
+  label: string;
+  icon?: IconComponent;
```

Comment:
is the IconComponent useful if we don't set it here?

Reply:
no, but i will need to update the `options` type in `front/src/modules/ui/input/components/Select.tsx`

Reply:
I think we should make it optional in the Select